src/main/java/ch/uzh/ifi/hase/soprafs21/service/LikeService.java

```diff
@@ -0,0 +1,64 @@
+package ch.uzh.ifi.hase.soprafs21.service;
+
+import ch.uzh.ifi.hase.soprafs21.entity.Like;
+import ch.uzh.ifi.hase.soprafs21.repository.LikeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.TransactionScoped;
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Service
+@Transactional
+public class LikeService {
+    private final LikeRepository likeRepository;
+
+    @Autowired
+    public LikeService(@Qualifier("likeRepository") LikeRepository likeRepository) {
+        this.likeRepository = likeRepository;
+    }
+    public List<Like> getAllLikes(){
+        return this.likeRepository.findAll();
+    }
+
+    public Like createLike(Like likeInput) {
```

Comment:
if a like already exists between the two ids a match should be crated

Reply:
I'm working on that